app/models/induction_record.rb

```diff
@@ -122,7 +141,7 @@ def matches_school_appropriate_body?
   end
 
   def transferring_in?
-    active_induction_status? && start_date > Time.zone.now
+    active_induction_status? && start_date > Time.zone.now && school_transfer
```

Comment:
I'm wondering if this didn't originally have the `school_transfer` bit so that it included newly qualified teachers starting at the school not just transfers from another school, but I'm not sure (or whether it is used anywhere)?

Reply:
I see this is only used in this request code.

I have added the `& school_transfer` bit to match the method exactly to the existing scope: 
`  scope :transferring_in, -> { active_induction_status.merge(start_date_in_future.and(school_transfer)) }
`

Reply:
Thanks for checking 👍